.github/workflows/knip.yml

```diff
@@ -20,11 +18,17 @@ jobs:
       - name: Install toolchain
         run: yarn install
 
-      - name: Report knip results to pull request
-        uses: Codex-/knip-reporter@v2
+      - name: Store PR number
+        run: echo ${{ github.event.number }} > pr-number.txt
+
+      - name: Run Knip
+        run: yarn knip || yarn -s knip --reporter json > knip-results.json
```

Comment:
Just curious about why the || with that command?

Reply:
Good question @gentlementlegen : `yarn knip`shows a human-readable result from Knip. If return code is `0` it means no errors were found, and the part after '||' will not be executed.

You can verify this with executing `yarn knip || echo "This should not show up if no errors"` command.
If errors are found, the second part generates output in machine-readable JSON format which is then consumed by knip-reporter workflow.